sources/packages/api/src/route-controllers/assessment/models/assessment.dto.ts

```diff
@@ -59,5 +59,29 @@ export class AssessmentNOAAPIOutDTO {
   offeringStudyStartDate: string;
   offeringStudyEndDate: string;
   msfaaNumber: string;
-  disbursement: any;
+  disbursement: DynamicAwardDTO;
+}
+
+export class AwardDetailsAPIOutDTO {
+  applicationNumber: string;
+  institutionName: string;
+  offeringIntensity: OfferingIntensity;
+  offeringStudyStartDate: string;
+  offeringStudyEndDate: string;
+  @ApiProperty({
+    description:
+      "Dynamic output of the workflow calculation. " +
+      "Contains data that could represent a part-time or a full-time award details. ",
+  })
+  estimatedAward: DynamicAwardDTO;
```

Comment:
This can be declared also as `estimatedAward: Record<string, string | number>;`, if I am not wrong.